form-flow-valtimo/src/main/kotlin/com/ritense/valtimo/formflow/FormFlowProcessLinkTaskProvider.kt

```diff
@@ -82,7 +84,7 @@ class FormFlowProcessLinkTaskProvider(
     private fun getFormFlowDefinition(formFlowId: String): FormFlowDefinition {
         val formFlowIdAsArray = formFlowId.split(":")
         if (formFlowIdAsArray.size != 2) {
-            throw IllegalArgumentException("Invalid Format found for formFlowId '${Strings.join(formFlowIdAsArray, ':' )}'. Form flow id must have format key:version")
+            throw IllegalArgumentException("Invalid Format found for formFlowId '${Strings.join(formFlowIdAsArray, ':')}'. Form flow id must have format key:version")
```

Comment:
maybe just put `$formFlowId` here instead of `${Strings.join(formFlowIdAsArray, ':')}`